cmd/goss/goss.go

```diff
@@ -357,30 +374,77 @@ func main() {
 					Name:  "kernel-param",
 					Usage: "add new goss kernel param",
 					Action: func(c *cli.Context) error {
+						fatalAlphaIfNeeded(c)
 						return goss.AddResources(c.GlobalString("gossfile"), "KernelParam", c.Args(), newRuntimeConfigFromCLI(c))
 					},
 				},
 				{
 					Name:  "mount",
 					Usage: "add new mount",
 					Action: func(c *cli.Context) error {
+						fatalAlphaIfNeeded(c)
 						return goss.AddResources(c.GlobalString("gossfile"), "Mount", c.Args(), newRuntimeConfigFromCLI(c))
 					},
 				},
 				{
 					Name:  "interface",
 					Usage: "add new interface",
 					Action: func(c *cli.Context) error {
+						fatalAlphaIfNeeded(c)
 						return goss.AddResources(c.GlobalString("gossfile"), "Interface", c.Args(), newRuntimeConfigFromCLI(c))
 					},
 				},
 			},
 		},
 	}
 
+	addAlphaFlagIfNeeded(app)
+	warnAlphaIfNeeded()
 	err := app.Run(os.Args)
 	if err != nil {
 		log.Fatal(err)
 	}
+	warnAlphaIfNeeded()
+}
+
+func addAlphaFlagIfNeeded(app *cli.App) {
+	if runtime.GOOS == "darwin" || runtime.GOOS == "windows" {
+		app.Flags = append(app.Flags, cli.StringFlag{
+			Name:   "use-alpha",
+			Usage:  fmt.Sprintf("goss is alpha-quality. Set to 1 to use anyway."),
```

Comment:
Clarify "goss for this platform is alpha.."

Reply:
Done in 17d2a39, please take a look?